etc/testing/lint.sh

```diff
@@ -18,7 +18,7 @@ done
 
 go get -u golang.org/x/lint/golint
 find "./src" \
-  \( -path "*.pb.go" -o "${skip_paths[@]}" \) -prune -o -name '*.go' -print \
+  \( -path "*.pb.go" "${skip_paths[@]}" \) -prune -o -name '*.go' -print \
```

Comment:
I made a slightly longer fix for this in #5434: https://github.com/pachyderm/pachyderm/pull/5434/files#diff-c645c5f5616b86e3827ef72cb6fe3815d38df279da2bcda31ec731155926debd

Hopefully git is smart enough to just merge them, since I think those changes are line-wise a superset of this change. But if you do get a merge conflict, that's what's going on.